### PR TITLE
Bump version to v2.0.0 & release notes

### PR DESCRIPTION
**Pull request by rsanchez15**

### CODE REVIEW

File: docs/rst/notes/notes.rst [hunk 1]
```diff
@@ -1,11 +1,47 @@
...
-.. TODO uncomment when there are forthcoming notes
-.. .. include:: forthcoming_version.rst
+This release includes the following **new features**:
+
+* Get IDL type definition of a topic.
+
+This release includes the following **updates**:
+
+* Update to Fast DDS v3.0.1.
+* Regenerate types with Fast DDS-Gen v4.1.0.
```
Comment (JesusPoderoso) [inline docs/rst/notes/notes.rst:13]:
```suggestion
* Regenerate types with Fast DDS-Gen v4.0.1.
```

File: docs/rst/notes/notes.rst
Comment (JesusPoderoso) [inline docs/rst/notes/notes.rst:1]: Version 1.1.0 notes should be removed from this file, and missing the 1.1.0 file reference:
``` md
.. include:: previous_versions/v1.1.0.rst
```

**Review by JesusPoderoso — CHANGES_REQUESTED** [review]

File: docs/rst/notes/notes.rst
Comment (rsanchez15) [reply docs/rst/notes/notes.rst:1]: File not saved locally. Addressed in e302db95b3c57866f5ff66c56142eb9ecd5f29d5

**Review by JesusPoderoso — APPROVED** [review]
LGTM

**Pull request merged**